_posts/en/newsletters/2024-07-19-newsletter.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 'Bitcoin Optech Newsletter #312'
+permalink: /en/newsletters/2024/07/19/
+name: 2024-07-19-newsletter
+slug: 2024-07-19-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Distributed key generation protocol for FROST:** Tim Ruffing and
+  Jonas Nick [posted][ruffing nick post] to the Bitcoin-Dev mailing list
+  a [BIP draft][chilldkg bip] with a [reference implementation][chilldkg
+  ref] of ChillDKG, a protocol for securely generating keys to be used
+  with [FROST][]-style [scriptless threshold signatures][topic threshold
+  signature] that are compatible with Bitcoin's [schnorr
+  signatures][topic schnorr signatures].
+
+  Scriptless threshold signatures are compatible with the creation of
+  `n` keys, any `t` of which can be used cooperatively to create a valid
+  signature.  For example, a 2-of-3 scheme creates three keys, any two
+  of which can produce a valid signature.  Being _scriptless_, the
+  scheme relies entirely on operations external to consensus and the
+  blockchain, unlike Bitcoin's built-in scripted threshold signature
+  operations (e.g., using `OP_CHECKMULTISIG`).
+
+  Similar to generating a regular Bitcoin private key, each person
+  creating a key for scriptless threshold signatures must generate
+  a large random-like number without disclosing that number to anyone
+  else.  However, each person must also distribute derived shares of
+  that number across the other users so that a threshold number of
+  them can create a signature if that key is unavailable.  Each user
+  must verify that the information they received from every other user
+  was generated correctly.  Several key generation protocols for
+  performing these steps exist, but they assume the key-generating users
+  have access to a communication channel that is encrypted and
+  authenticated between individual pairs of users and that also allows
+  uncensorable authenticated broadcast from each user to all other
+  users.  The _Chill distributed key generation_ (ChillDKG) protocol
+  combines a well-known key generation algorithm for FROST with
+  additional modern cryptographic primitives and simple algorithms to
+  provide the necessary secure, authenticated, and provably uncensored
+  communication.
+
+  Encryption and authentication between participants starts with an
+  [elliptic curve diffie-hellman][ecdh] (ECDH) exchange.  Proof of
+  non-censorship is obtained by each participant using their base key to
+  sign a transcript of the session from its start to its end, where the
+  scriptless threshold public key is produced.  Before accepting the
+  threshold public key as correct, each participant verifies every other
+  participant's signed session transcript.
+
+  The goal was to design a fully generalized protocol that will be
+  usable in all cases where people want to generate keys for
+  Bitcoin-style scriptless threshold signatures.  Additionally, the
```

Comment:
This is somewhat misleading. ChillDKG works for FROST, but it could be insecure when used in other signing schemes. See section ["Use ChillDKG only for FROST"](https://github.com/BlockstreamResearch/bip-frost-dkg?tab=readme-ov-file#use-chilldkg-only-for-frost). In short, this is because ChillDKG participants can bias the final key (which is fine for FROST).